### ISSUE TITLE
Distro broken after move to 3.0.0 Snapshots

### ISSUE THREAD

**Issue by kgoderis** [openhab/openhab-distro#1037]
Last moments of 2019 to be on the cutting edge of technology ;-)

I have done a clean install of the IDE, and made sure that all repo's are at the latest commit, in order to test out the IDE install itself, and the demo.app. Something is broken

First of all, the bndrun file needs to be altered slightly to remove the missing ui.paper and and the reference to transform.map (this bundle still references smart home).

Secondly, when starting the demo app, the runtime does not go further than :

```
Welcome to Apache Felix Gogo

g! 14:05:16.518 [pool-8-thread-1] INFO  o.o.c.service.AbstractWatchService:170 - Loading script 'demo.js'
```

Nothing else happens, even as all bundles are loaded :

```
    0|Active     |    0|org.eclipse.osgi (3.13.100.v20180827-1536)
    1|Active     |    1|ch.qos.logback.classic (1.2.3)
    2|Active     |    1|ch.qos.logback.core (1.2.3)
    3|Active     |    1|com.eclipsesource.jaxrs.jersey-all (2.22.2)
    4|Active     |    1|com.eclipsesource.jaxrs.publisher (5.3.1.201602281253)
    5|Active     |    1|com.google.gson (2.8.2.v20180104-1110)
    6|Active     |    1|com.google.inject (3.0.0)
    7|Active     |    1|javax.measure.unit-api (1.0.0)
    8|Active     |    1|joda-time (2.9.2)
    9|Active     |    1|jollyday (0.5.8)
   10|Active     |    1|log4j (1.2.17)
   11|Active     |    1|org.antlr.runtime (3.2.0.v201101311130)
   12|Active     |    1|org.apache.commons.exec (1.1.0)
   13|Active     |    1|org.apache.commons.fileupload (1.3.3)
   14|Active     |    1|org.apache.commons.io (2.2.0)
   15|Active     |    1|org.apache.commons.lang (2.6.0)
   16|Active     |    1|org.apache.felix.configadmin (1.9.8)
   17|Active     |    1|org.apache.felix.gogo.command (1.0.2)
   18|Active     |    1|org.apache.felix.gogo.runtime (1.0.10)
   19|Active     |    1|org.apache.felix.gogo.shell (1.0.0)
   20|Active     |    1|org.apache.felix.http.servlet-api (1.1.2)
   21|Active     |    1|org.apache.felix.inventory (1.0.4)
   22|Active     |    1|org.apache.felix.log (1.2.0)
   23|Active     |    1|org.apache.felix.logback (1.0.0)
   24|Active     |    1|org.apache.felix.scr (2.1.10)
   25|Active     |    1|org.apache.felix.webconsole (4.3.4)
   26|Active     |    1|org.apache.felix.webconsole.plugins.ds (2.0.8)
   27|Active     |    1|org.apache.servicemix.bundles.xstream (1.4.7.1)
   28|Active     |    1|org.apache.xbean.bundleutils (4.12.0)
   29|Active     |    1|org.apache.xbean.finder (4.12.0)
   30|Active     |    1|org.eclipse.emf.common (2.12.0.v20160420-0247)
   31|Active     |    1|org.eclipse.emf.ecore (2.12.0.v20160420-0247)
   32|Active     |    1|org.eclipse.emf.ecore.xmi (2.12.0.v20160420-0247)
   33|Active     |    1|org.eclipse.equinox.event (1.4.300.v20180827-1235)
   34|Active     |    1|org.eclipse.equinox.metatype (1.4.500.v20180827-1235)
   35|Active     |    1|org.glassfish.hk2.external.aopalliance-repackaged (2.4.0.b34)
   36|Active     |    1|org.mapdb.mapdb (1.0.9)
   37|Active     |    1|org.osgi.service.event (1.4.0.201802012106)
   38|Active     |    1|org.osgi.service.metatype (1.4.0.201802012106)
   39|Active     |    1|org.threeten.extra (1.4.0)
   40|Active     |    1|slf4j.api (1.7.25)
   41|Active     |    1|tec.uom.lib.uom-lib-common (1.0.3)
   42|Active     |    1|tec.uom.se (1.0.10)
   43|Active     |    1|com.google.guava (27.1.0.jre)
   44|Active     |    1|com.google.guava.failureaccess (1.0.1)
   45|Active     |    1|io.github.classgraph (4.8.35.v20190528-1517)
   46|Active     |    1|org.eclipse.equinox.common (3.10.400.v20190516-1504)
   47|Active     |    1|org.eclipse.xtend.lib (2.19.0.v20190902-0728)
   48|Active     |    1|org.eclipse.xtend.lib.macro (2.19.0.v20190902-0728)
   49|Active     |    1|org.eclipse.xtext (2.19.0.v20190902-0737)
   50|Active     |    1|org.eclipse.xtext.common.types (2.19.0.v20190902-0952)
   51|Active     |    1|org.eclipse.xtext.util (2.19.0.v20190902-0737)
   52|Active     |    1|org.eclipse.xtext.xbase (2.19.0.v20190902-0952)
   53|Active     |    1|org.eclipse.xtext.xbase.lib (2.19.0.v20190902-0728)
   54|Active     |    1|org.objectweb.asm (7.1.0)
   55|Active     |    1|org.objectweb.asm.commons (7.1.0)
   56|Active     |    1|org.objectweb.asm.tree (7.1.0)
   57|Active     |    1|org.eclipse.jetty.client (9.4.20.v20190813)
   58|Active     |    1|org.eclipse.jetty.http (9.4.20.v20190813)
   59|Active     |    1|org.eclipse.jetty.io (9.4.20.v20190813)
   60|Active     |    1|org.eclipse.jetty.proxy (9.4.20.v20190813)
   61|Active     |    1|org.eclipse.jetty.security (9.4.20.v20190813)
   62|Active     |    1|org.eclipse.jetty.server (9.4.20.v20190813)
   63|Active     |    1|org.eclipse.jetty.servlet (9.4.20.v20190813)
   64|Active     |    1|org.eclipse.jetty.util (9.4.20.v20190813)
   65|Active     |    1|org.eclipse.jetty.websocket.api (9.4.20.v20190813)
   66|Active     |    1|org.eclipse.jetty.websocket.client (9.4.20.v20190813)
   67|Active     |    1|org.eclipse.jetty.websocket.common (9.4.20.v20190813)
   68|Active     |    1|org.eclipse.jetty.xml (9.4.20.v20190813)
   69|Active     |    1|org.ops4j.pax.swissbox.optional.jcl (1.8.3)
   70|Active     |    1|org.ops4j.pax.web.pax-web-api (7.2.11)
   71|Active     |    1|org.ops4j.pax.web.pax-web-jetty (7.2.11)
   72|Active     |    1|org.ops4j.pax.web.pax-web-runtime (7.2.11)
   73|Active     |    1|org.ops4j.pax.web.pax-web-spi (7.2.11)
   74|Active     |    1|org.openhab.core (3.0.0.201912311258)
   75|Active     |    1|org.openhab.core.audio (3.0.0.201912311259)
   76|Active     |    1|org.openhab.core.automation (3.0.0.201912311258)
   77|Active     |    1|org.openhab.core.automation.module.media (3.0.0.201912311300)
   78|Active     |    1|org.openhab.core.automation.module.script (3.0.0.201912311300)
   79|Active     |    1|org.openhab.core.automation.module.script.rulesupport (3.0.0.201912311300)
   80|Active     |    1|org.openhab.core.automation.rest (3.0.0.201912311300)
   81|Active     |    1|org.openhab.core.binding.xml (3.0.0.201912311300)
   82|Active     |    1|org.openhab.core.boot (3.0.0.201912311300)
   83|Active     |    1|org.openhab.core.config.core (3.0.0.201912311258)
   84|Active     |    1|org.openhab.core.config.discovery (3.0.0.201912311258)
   85|Active     |    1|org.openhab.core.config.dispatch (3.0.0.201912311300)
   86|Active     |    1|org.openhab.core.config.xml (3.0.0.201912311300)
   87|Active     |    1|org.openhab.core.ephemeris (3.0.0.201912311258)
   88|Active     |    1|org.openhab.core.id (3.0.0.201912311300)
   89|Active     |    1|org.openhab.core.io.console (3.0.0.201912311258)
   90|Active     |    1|org.openhab.core.io.console.rfc147 (3.0.0.201912311300)
   91|Active     |    1|org.openhab.core.io.http (3.0.0.201912311259)
   92|Active     |    1|org.openhab.core.io.monitor (3.0.0.201912311300)
   93|Active     |    1|org.openhab.core.io.net (3.0.0.201912311258)
   94|Active     |    1|org.openhab.core.io.rest (3.0.0.201912311300)
   95|Active     |    1|org.openhab.core.io.rest.core (3.0.0.201912311300)
   96|Active     |    1|org.openhab.core.io.rest.optimize (3.0.0.201912311300)
   97|Active     |    1|org.openhab.core.io.rest.sitemap (3.0.0.201912311300)
   98|Active     |    1|org.openhab.core.io.rest.sse (3.0.0.201912311300)
   99|Active     |    1|org.openhab.core.model.core (3.0.0.201912311258)
  100|Active     |    1|org.openhab.core.model.item (3.0.0.201912311300)
  101|Active     |    1|org.openhab.core.model.item.runtime (3.0.0.201912311300)
  102|Active     |    1|org.openhab.core.model.persistence (3.0.0.201912311259)
  103|Active     |    1|org.openhab.core.model.persistence.runtime (3.0.0.201912311300)
  104|Active     |    1|org.openhab.core.model.rule (3.0.0.201912311259)
  105|Active     |    1|org.openhab.core.model.rule.runtime (3.0.0.201912311259)
  106|Active     |    1|org.openhab.core.model.script (3.0.0.201912311259)
  107|Active     |    1|org.openhab.core.model.script.runtime (3.0.0.201912311300)
  108|Active     |    1|org.openhab.core.model.sitemap (3.0.0.201912311300)
  109|Active     |    1|org.openhab.core.model.sitemap.runtime (3.0.0.201912311300)
  110|Active     |    1|org.openhab.core.model.thing (3.0.0.201912311300)
  111|Active     |    1|org.openhab.core.model.thing.runtime (3.0.0.201912311300)
  112|Active     |    1|org.openhab.core.persistence (3.0.0.201912311258)
  113|Active     |    1|org.openhab.core.scheduler (3.0.0.201912311258)
  114|Active     |    1|org.openhab.core.semantics (3.0.0.201912311300)
  115|Active     |    1|org.openhab.core.storage.mapdb (3.0.0.201912311300)
  116|Active     |    1|org.openhab.core.thing (3.0.0.201912311258)
  117|Active     |    1|org.openhab.core.thing.xml (3.0.0.201912311300)
  118|Active     |    1|org.openhab.core.transform (3.0.0.201912311259)
  119|Active     |    1|org.openhab.core.ui (3.0.0.201912311300)
  120|Active     |    1|org.openhab.core.ui.icon (3.0.0.201912311300)
  121|Active     |    1|org.openhab.core.voice (3.0.0.201912311259)
  122|Active     |    1|org.openhab.ui.basic (3.0.0.201912311248)
  123|Active     |    1|org.openhab.ui.dashboard (3.0.0.201912311248)
  124|Active     |    1|org.openhab.ui.iconset.classic (3.0.0.201912311249)
  125|Active     |    1|org.openhab.ui.restdocs (3.0.0.201912311249)
  126|Active     |    1|org.apache.servicemix.specs.activation-api-1.1 (2.9.0)
  127|Active     |    1|org.apache.servicemix.specs.annotation-api-1.3 (1.3.0.1)
  128|Active     |    1|org.apache.servicemix.specs.jaxb-api-2.2 (2.9.0)
  129|Active     |    1|org.apache.servicemix.specs.stax-api-1.2 (2.9.0)
```

**Comment by kgoderis** [comment]
Solved. smarthome.cfg is still using org.smarthome.core references

**Comment by cweitkamp** [comment]
This should be solved with #1030.

**Comment by kaikreuzer** [comment]
3.0 Distro is pretty nice again :-)
